src/js/edu-benefits/pages/applicantService.js

```diff
@@ -26,7 +26,18 @@ export default function applicantServicePage() {
         },
         'ui:required': form => _.get('view:applicantServed', form),
         items: {
-          serviceStatus: { 'ui:title': 'Type of separation or discharge' }
+          serviceStatus: {
+            'ui:title': 'Type of separation or discharge',
+            'ui:options': {
+              updateSchema: (pageData, formData, currentSchema, index) => {
+                return {
+                  minLength: _.get(`applicantService.data.toursOfDuty[${index}].serviceBranch`, formData) === 'Army'
```

Comment:
Isn't `serviceBranch` a text field? We should probably `.toLowerCase()` it and compare it to `'army'`, aye?